plugins/thunderbolt/fu-thunderbolt-device.c

```diff
@@ -580,6 +580,8 @@ fu_thunderbolt_device_setup_retimer(FuDevice *device, GError **error)
 	fu_device_add_flag(device, FWUPD_DEVICE_FLAG_DUAL_IMAGE);
 	fu_device_add_flag(device, FWUPD_DEVICE_FLAG_INTERNAL);
 	fu_device_add_internal_flag(device, FU_DEVICE_INTERNAL_FLAG_NO_AUTO_REMOVE);
+	fu_device_add_flag(device, FWUPD_DEVICE_FLAG_SKIPS_RESTART);
```

Comment:
is this true of all tbt devices, or just retimers?

Reply:
this is needed just for re timers

Reply:
But this is going to be set for all thunderbolt devices, no? i.e. not just retimers.

Reply:
Ohh, hmm. This is because FuThunderboltDevice is being used for both rather than splitting them up into two different GTypes. I think given the functionality is diverging we probably want to do that sooner rather than later.

Reply:
I'll work on a WIP to do the split.